app/views/cakes/index.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= render "flash" %>
```

Comment:
This shouldn't be here - we want to display it for every view. We can avoid putting it to every template file ;) Please check how :)